.circleci/config.yml

```diff
@@ -20,6 +25,14 @@ postgres_service: &postgres_service
 mongo_service: &mongo_service
   image: mongo
 
+mysql_service: &mysql_service
+  image: circleci/mysql:5.7
+  environment:
+    MYSQL_ROOT_PASSWORD: rootpw
+    MYSQL_DATABASE: circle_database
```

Comment:
This is defined in two places, is it intentional?

Reply:
Yes. One is the ENV var that the container uses to create the database, the other is for our running test to connect to it.